Assets/Packages/com.studio23.ss2.buttoniconresourcemanager/Runtime/Data/KeyIcons.cs

```diff
@@ -34,15 +33,13 @@ public Sprite GetIcon(Platform platform)
 
 		public Sprite GetIcon(string platformName)
 		{
-			switch (platformName)
-			{
-				case "Xbox":
-					return _xboxIcon;
-				case "Playstation":
-					return _playstationIcon;
-				default:
-					return null;
-			}
+			string platform = platformName.ToLower();
+			if (platform.Contains("playstation") || platform.Contains("dual"))
+				return _playstationIcon;
+			else if (platform.Contains("xbox") || platform.Contains("xinput") || platform.Contains("gamepad"))
+				return _xboxIcon;
+			else
+				return null;
```

Comment:
Although this block allows for the check of specific keywords, it introduces the potential for missing edge cases. There are many third-party controllers on the market that are not directly from PlayStation or Xbox and these may have different names that don't match with keywords in the checks. So by default, it should return Xbox icons when nothing matches. Another keyboard that may need to be added is "controller". This change should be included.

Reply:
This edge case has been noted and is added to issues for a fix in the next update.